pkgs/development/python-modules/torchaudio/default.nix

```diff
@@ -31,27 +34,42 @@ buildPythonPackage rec {
     TORCH_CUDA_ARCH_LIST = "${lib.concatStringsSep ";" torch.cudaCapabilities}";
   };
 
+  # https://github.com/pytorch/audio/blob/v2.1.0/docs/source/build.linux.rst#optional-build-torchaudio-with-a-custom-built-ffmpeg
+  FFMPEG_ROOT = symlinkJoin {
+    name = "ffmpeg";
+    paths = [
+      ffmpeg-full.bin
+      ffmpeg-full.dev
+      ffmpeg-full.lib
+    ];
+  };
+
```

Comment:
This should declared in a `let` binding to avoid passing it as an argument to the builder.

Reply:
Not sure I get what you want me to do.
I added it to a `let` binding at the top of the derivation, but I don't see how it changes anything.

Reply:
I apologize, I must have misread the file when I last reviewed it because I thought this was declared and then referenced as a `nativeBuildInput` or `buildInput`.

You're correct; the `let` binding here doesn't matter -- the goal is to set the environment variable `FFMPEG_ROOT` (per the comment you have) and as it isn't referenced elsewhere in the expression, a `let` binding doesn't help.

Reply:
Ok, so I ended up putting it path in the core of the derivation (without the `let` `in` block).
Is that alright for you ?